### PR TITLE
iussue #5: fix the C# name convention violation

### PR DESCRIPTION
**Pull request by gaufung**

### CODE REVIEW

File: Src/Tracers/Zipkin/V2/V1SpanConverter.cs [hunk 1]
```diff
@@ -42,66 +42,66 @@ public static V1SpanConverter Create()
         public List<Span> Convert(zipkin4net.Tracers.Zipkin.Span source)
         {
             List<Span> results = new List<Span>();
-            convert(source, results);
+            Convert(source, results);
             return results;
         }
 
-        private void convert(zipkin4net.Tracers.Zipkin.Span source, List<Span> sink)
+        private void Convert(zipkin4net.Tracers.Zipkin.Span source, List<Span> sink)
```
Comment (observeralone) [inline Src/Tracers/Zipkin/V2/V1SpanConverter.cs:49]: According to the naming convention, the first letter of a `private` method should be lowercase. Please pay close attention.

**Comment by observeralone** [comment]
According to the naming convention, the first letter of a `private` method should be lowercase. Please take a close look. 
I will close this `issue`, or you can close it yourself unless you can provide a more convincing reason.

File: Src/Tracers/Zipkin/V2/V1SpanConverter.cs (hunk 1 above)
Comment (gaufung) [reply Src/Tracers/Zipkin/V2/V1SpanConverter.cs:49]: No. all methods in C# should be upper case with any modifier. 

https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions

Comment (observeralone) [reply Src/Tracers/Zipkin/V2/V1SpanConverter.cs:49]: > No. all methods in C# should be PascalCasing with any modifier.
> 
> https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions

Maybe you are right, although I don't think making `public` and `private` naming conventions consistent is a good choice. However, I respect everyone's decision.

**Pull request merged**